src/imgix-core-js.js

```diff
@@ -237,23 +231,50 @@
         return srcset.slice(0,-2);
     };
 
+    function validateAndDestructureOptions(options) {
+      if (options.widthTolerance !== undefined) {
+        validateWidthTolerance(options.widthTolerance);
+        widthTolerance = options.widthTolerance;
+      } else {
+        widthTolerance = DEFAULT_SRCSET_WIDTH_TOLERANCE;
+      }
+
+      if (options.minWidth !== undefined && options.maxWidth !== undefined) {
+        minWidth = options.minWidth;
+        maxWidth = options.maxWidth;
+        validateRange(minWidth, maxWidth);
+      } else if (options.minWidth !== undefined) {
+        minWidth = options.minWidth;
+        maxWidth = MAX_SRCSET_WIDTH;
+        validateRange(minWidth, maxWidth);
+      } else if (options.maxWidth !== undefined) {
+        minWidth = MIN_SRCSET_WIDTH;
+        maxWidth = options.maxWidth;
+        validateRange(minWidth, maxWidth);
+      } else {
+        minWidth = MIN_SRCSET_WIDTH;
+        maxWidth = MAX_SRCSET_WIDTH;
+      }
+
+      return [widthTolerance, minWidth, maxWidth];
+    };
+
     function validateRange(min, max) {
-      if (!(Number.isInteger(min) && Number.isInteger(max)) || (min < 0 || max < 0)) {
+      if (!(Number.isInteger(min) && Number.isInteger(max)) || (min <= 0 || max <= 0)) {
```

Comment:
```suggestion
      if (!(Number.isInteger(min) && Number.isInteger(max)) || (min <= 0 || max <= 0) || min < max) {
```

Reply:
Can we add a test for this new case as well?

Reply:
I think you mean `min > max` ?